cluster/gce/container-linux/configure-helper.sh

```diff
@@ -284,6 +284,7 @@ clusters:
 - name: local
   cluster:
     certificate-authority-data: ${KUBELET_CA_CERT}
+    server: https://example:443
```

Comment:
can we not include the correct hostname here, rather than fixing up at use time?